middleware/proxy/metrics.go

```diff
@@ -12,7 +12,7 @@ import (
 var (
 	RequestDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Namespace: middleware.Namespace,
-		Subsystem: subsystem,
+		Subsystem: "proxy",
 		Name:      "request_duration_milliseconds",
 		Buckets:   append(prometheus.DefBuckets, []float64{50, 100, 200, 500, 1000, 2000, 3000, 4000, 5000, 10000}...),
```

Comment:
Ahh, I see. So this just adds longer time buckets since it's often going over the Internet in the proxy, I suppose.